virttest/test_setup.py

```diff
@@ -1313,6 +1312,31 @@ def get_pf_mlx(self):
             cmd, shell=True, timeout=60).splitlines()
         return (PF_devices)
 
+    def assign_ip_pf_mlx(self):
+        """
+        Set the static IP for the PF devices for mlx5_core driver
+        """
+        # This function assigns static IP for the PF device of
+        # mellanox cx4 card
+        set_ip = 0
+        pf_devices = self.get_pf_mlx()
```

Comment:
This looks redundant, we call this function again in L1494, lets pass it as parameter and use it instead calling twice.